server/routes/offenceRoutes.ts

```diff
@@ -820,7 +751,7 @@ export default class OffenceRoutes {
       errors: req.flash('errors') || [],
       backLink: submitToEditOffence
         ? `/person/${nomsId}/${addOrEditCourtCase}/${courtCaseReference}/${addOrEditCourtAppearance}/${appearanceReference}/offences/${offenceReference}/edit-offence`
-        : `/person/${nomsId}/${addOrEditCourtCase}/${courtCaseReference}/${addOrEditCourtAppearance}/${appearanceReference}/offences/${offenceReference}/offence-date`,
```

Comment:
this old back link was wrong I think.. it should have gone back to the terror screen from sentence-type